### ISSUE TITLE
Deep Learning Book Citation

### ISSUE THREAD

**Issue by cgreene** [greenelab/deep-review#387]
I used a bare URL in #382. We should fix this to a relevant citation form before using it.

**Comment by dhimmel** [comment]
This issue refers to `[@url:http://www.deeplearningbook.org/]`. The [webpage](http://www.deeplearningbook.org/) provides the following bibtex:

```bibtex
@book{Goodfellow-et-al-2016,
    title={Deep Learning},
    author={Ian Goodfellow and Yoshua Bengio and Aaron Courville},
    publisher={MIT Press},
    note={\url{http://www.deeplearningbook.org}},
    year={2016}
}
```

Given that this book doesn't have an ISBN or DOI, our metadata options depend on #381.

**Comment by gwaygenomics** [comment]
The book has since been [published](https://mitpress.mit.edu/books/deep-learning) with ISBN 9780262337434

**Comment by dhimmel** [comment]
Interestingly, the ISBN doesn't seem to have metadata on the two services I could find:

+ http://isbndb.com/search/all?query=9780262337434
+ http://xisbn.worldcat.org/webservices/xid/isbn/9780262337434?method=getMetadata&format=xml&fl=*

More discussion on book metadata [here](http://stackoverflow.com/questions/3287433/how-to-get-book-metadata).

**Comment by agapow** [comment]
My impression of worldcat and isbndb is that there's quite a lag before titles get listed (and they may never get listed). Not a solution, but we probably need another way.

**Comment by agitter** [comment]
Closing this. I manually fixed it in #515
